src/components/VInput/VInput.vue

```diff
@@ -104,6 +108,8 @@
         @blur.once="dirty = true"
         v-on="listeners"
       />
+
+      <slot name="label" />
```

Comment:
Feels like this should be replacing the label text above the input. Not adding a slot after the input.

Reply:
I've moved it next to the label `'vts-input__text'`